docs/source/use/pathways.md

```diff
@@ -0,0 +1,100 @@
+# Four ways to use `repo2docker` images in a JupyterHub
+
+Many users of `repo2docker` primarily wish to define the environment for a Binder or a JupyterHub. Here are four ways to accomplish this, from least-to-most work.
+
+```{figure} ../_static/images/whentouse.svg
```

Comment:
I find this figure rather confusing, as there's no obvious start or end point, but making a more intuitive diagram is perhaps for a separate issue. Fortunately, the prose that follows is quite clear.

Reply:
I'm adding a more standard decision tree structure in the next commit that should clarify it a bit